src/main/java/org/candlepin/subscriptions/product/OfferingWorker.java

```diff
@@ -57,4 +68,18 @@ public void receive(OfferingSyncTask task) {
 
     controller.syncOffering(sku);
   }
+
+  @JmsListener(destination = "#{@umbProperties.productTopic}")
+  public void receive(String productMessageXml) throws JsonProcessingException {
+    log.debug("Received message from UMB offering product{}", productMessageXml);
+    if (!umbProperties.isProcessingEnabled()) {
+      return;
```

Comment:
Feels like there should be some logging that happens here. Would also be nice to have a test case around this logic.

Reply:
Added test for the calling method and changed this method to private

Reply:
Please add a debug message for when UMB processing is not enabled. Could be helpful in cases where messages are being skipped because of this setting.

Reply:
Done